patches/core/0025-span-fallback-to-gsl-span-if-std-span-not-exists.patch

```diff
@@ -0,0 +1,207 @@
+From 2cab7f70515a8fa3ca488c937e5540e02661835d Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?=C4=90o=C3=A0n=20Tr=E1=BA=A7n=20C=C3=B4ng=20Danh?=
+ <congdanhqx@gmail.com>
+Date: Sun, 1 Nov 2020 13:01:48 +0700
+Subject: [PATCH] span: fallback to gsl::span if std::span not exists
```

Comment:
Also a GCC incompatibility or why is this necessary?

Reply:
Yes, gcc 9.3.0 hasn't ship `std::span` yet, it's a C++20 feature. It should be fine with gcc 10.